### PR TITLE
Fixed installation guide command typo

### PR DESCRIPTION
**Pull request by andrew-fennell**
## PR Summary
I'm think `python -mpip ...` is supposed to be `python -m pip ...` in both of the cases in the installation guide.

## PR Checklist

**Documentation**
- [N/A] New features are documented, with examples if plot related.
- [N/A] New features have an entry in `doc/users/next_whats_new/` (follow instructions in README.rst there).
- [N/A] API changes documented in `doc/api/next_api_changes/` (follow instructions in README.rst there).
- [ ] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).

### CODE REVIEW

**Comment by andrew-fennell** [comment]
I also noticed that the installation document uses `python` for everything, but the entire "Checking your installation" section uses `python3`.

Is this intentional? If not, I can standardize that as apart of this PR.

https://matplotlib.org/stable/users/installing/index.html?highlight=visual%20studio#checking-your-installation

File: doc/users/installing/index.rst [hunk 1]
```diff
@@ -281,11 +281,11 @@ be manually installed on Macports with ::
 Once pip is installed, you can install Matplotlib and all its dependencies with
 from the Terminal.app command line::
 
-   python3 -mpip install matplotlib
+   python3 -m pip install matplotlib
 
 (``sudo python3.6 ...`` on Macports).
```
Comment (oscargus) [inline doc/users/installing/index.rst:286]: Some Mac person (@greglucas I have identified as one) can probably tell if there is a better thing than `3.6` to write here.

Comment (oscargus) [reply doc/users/installing/index.rst:286]: I now realize that the whole section is about OSX, so probably I should stop reviewing... 

But my, not very qualified, guess is that you are doing the right thing.

Comment (oscargus) [reply doc/users/installing/index.rst:286]: After reading up a bit:

* Both `-mpip` and `-m pip` works, but consistency is nice.
* OSX comes with 2.7 as default `python` so for the OSX parts it should be `python3`.

Comment (andrew-fennell) [reply doc/users/installing/index.rst:286]: I didn't realize -mpip was valid. 

In that case, maintainers can decide whether consistency warrants a change. I'm OK with it either way.

@oscargus Thank you for looking into both of those issues!

Comment (timhoffm) [reply doc/users/installing/index.rst:286]: Let's always use `-m pip`. While I personally always leave out the space when typing, separating the parameter and it's argument is more clear.

**Review by timhoffm — APPROVED** [review]

**Comment by timhoffm** [comment]
For simplicity, I merge this as is. If there is interest in sorting out the `python3` topic, that can be addressed separately.

**Pull request merged**